pages/ksphere/konvoy/1.5.0-beta/install/install-secured/index.md

```diff
@@ -61,6 +61,21 @@ For all hosts that are part of the cluster -- except the **deploy host** -- you
 On highly secured clusters you may need to modify the `cluster.yaml` file with additional options.
 See the sample file below for possible changes that may be applied in your cluster.
 
+## Kubernetes CVE Patches
+
+At times, CVEs may be discovered in the Kubernetes codebase. Based on the severity and the impact of a specific CVE, you may want to temporarily use alternative docker images for the core Kubernetes components instead of the default `k8s.gcr.io` repository.
```

Comment:
```suggestion
At times, CVEs can be discovered in the Kubernetes codebase. Based on the severity and impact of a specific CVE, you can temporarily use alternative docker images for the core Kubernetes components instead of the default `k8s.gcr.io` repository.
```